src/ota.rs

```diff
@@ -239,6 +300,11 @@ impl EspOta {
         Ok(Self(()))
     }
 
+    /// Returns the currently configured boot slot.
```

Comment:
boot slot that will go into effect on the next reboot.

Reply:
Thanks. I feel like this is already implied, i.e. "boot slot" will obviously be used for the next boot. Or am I missing something?

Reply:
No I think you're right